pages/18f/how-18f-works/research-guidelines.md

```diff
@@ -106,6 +106,13 @@ participants, especially from underserved communities, and helps build trust.
 Without providing incentives and compensation, we are likely to hear only from
 the people who can afford to volunteer their time.
 
+## Planning for compensation
```

Comment:
I wonder if planning for compensation actually includes the checklist? Getting a giftbit account is seems like part of planning as I read it now.